lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -162,6 +162,14 @@ def elb_client(region = default_region)
         self.class.elb_client(region)
       end
 
+      def self.elbv2_client(region = default_region)
```

Comment:
@MWilkinson Have the elbv2 resource taken form yet?  I'm in the middle of adding some code to the elb resource, and oh my, I'm realizing how much needs to be improved.